.gitignore

```diff
@@ -0,0 +1,7 @@
+.vscode/settings.json
+.vscode/panel.code-workspace
+__pycache__/dodo.cpython-37.pyc
+__pycache__/setup.cpython-37.pyc
+0.5.0
+panel/.version

```

Comment:
Patterns which are specific to a particular repository but which do not need to be shared with other related repositories (e.g., auxiliary files that live inside the repository but are specific to one user’s workflow) should go into the $GIT_DIR/info/exclude file.
https://git-scm.com/docs/gitignore

Reply:
I'm going to be adding a gitignore myself shortly so let's drop this one for now.